versions.properties

```diff
@@ -130,483 +102,30 @@ version.io.jsonwebtoken..jjwt=0.9.1
 
 version.google.dagger=2.46.1
 
-version.google.android.material=1.4.0
-##                  # available=1.5.0-alpha01
-##                  # available=1.5.0-alpha02
-##                  # available=1.5.0-alpha03
-##                  # available=1.5.0-alpha04
-##                  # available=1.5.0-alpha05
-##                  # available=1.5.0-beta01
-##                  # available=1.5.0-rc01
-##                  # available=1.5.0
-##                  # available=1.6.0-alpha01
-##                  # available=1.6.0-alpha02
-##                  # available=1.6.0-alpha03
-##                  # available=1.6.0-beta01
-##                  # available=1.6.0-rc01
-##                  # available=1.6.0
-##                  # available=1.6.1
-##                  # available=1.7.0-alpha01
-##                  # available=1.7.0-alpha02
-##                  # available=1.7.0-alpha03
-##                  # available=1.7.0-beta01
-##                  # available=1.7.0-rc01
-##                  # available=1.7.0
-##                  # available=1.8.0-alpha01
-##                  # available=1.8.0-alpha02
-##                  # available=1.8.0-alpha03
-##                  # available=1.8.0-beta01
-##                  # available=1.8.0-rc01
-##                  # available=1.8.0
-##                  # available=1.9.0-alpha01
-##                  # available=1.9.0-alpha02
-##                  # available=1.9.0-beta01
-##                  # available=1.9.0-rc01
-##                  # available=1.9.0
+version.google.android.material=1.9.0
```

Comment:
In my $workProject, minor version bumps of material library usually cause screenshot-tests to fail 🤷 I'm not yet sure what's covered, but I won't be surprised if this change fails some screenshots